.travis.yml

```diff
@@ -8,11 +8,15 @@ rvm:
   - 2.2
   - 2.3.3
   - 2.4.0
+  - ruby-head
 
 matrix:
   include:
     - rvm: 2.3.0
       script: bundle exec rake test_isolated
+  allow_failures:
+    - rvm: ruby-head
+  fast_finish: true
```

Comment:
@rafaelfranca @matthewd I added allow_failures code again after @matthewd 's comment.
I also update rubies to latest version in `.travis.yml`.